interface/java_binding/src/main/java/org/khronos/ktx/KtxTexture2.java

```diff
@@ -45,4 +45,30 @@ public static native KtxTexture2 createFromNamedFile(String filename,
     public static KtxTexture2 createFromNamedFile(String filename) {
         return createFromNamedFile(filename, KtxTextureCreateFlagBits.LOAD_IMAGE_DATA_BIT);
     }
+
+    /**
+     * Deflate the data in a {@link KtxTexture2} object using Zstandard.
+     *
+     * The texture's levelIndex, dataSize, DFD  and supercompressionScheme will
```

Comment:
The data pointer is also updated. Looks like an oversight in the documentation probably emanating from the native documentation. Please update the doc here and in native, if indeed it does not mention this.

Reply:
This was taken from https://github.com/KhronosGroup/KTX-Software/blob/5414bd0ebfdfdf4ecb8c613a6edb21372303c2b9/lib/writer2.c#L762 .

I have to rely on the documentation to be complete and correct when "porting stuff". 

But I have taken an note to update the documentation here (in the native layer, which will probably affect _all_ documentations in the "bindings"...)

Reply:
> I have to rely on the documentation to be complete and correct when "porting stuff".

Absolutely. Sorry for the omission.

Reply:
Please fix the documentation in this PR. The overhead of a PR is too big to justify a separate one for these tiny documentation fixes.

Reply:
Still need the documentation fix.

Reply:
I can add this documentation fix here

But note the first bullet point of the things "Not (yet) supposed to be addressed here", at the bottom of https://github.com/KhronosGroup/KTX-Software/pull/886#issue-2208337582 :   

The corresponding functions do not yet (all) exist in the bindings. For example, the Python binding has `deflateZstd`, but not `deflateZLIB`. So this will have to be aligned at a later point anyhow.

Reply:
The documentation has been fixed as of https://github.com/KhronosGroup/KTX-Software/pull/879/commits/d3dd37d9f588470e6184beb651052b6a1432448e and https://github.com/KhronosGroup/KTX-Software/pull/879/commits/9a0b280dbb1d04f2d11f9fb7843bdaae2a3385e2